avm/res/automation/automation-account/main.bicep

```
@@ -25,6 +25,15 @@ param credentials credentialType = []
 @description('Optional. List of modules to be created in the automation account.')
 param modules array = []
 
+@description('Optional. List of powershell72 modules to be created in the automation account.')
+param powershell72Modules pwsh72ModulesType
```

For this and other types, please keep the convention <parameteranameInSingularForm>Type. Not documented, at least not yet, but consistent with the schema of other shared parameters, e.g., diagnosticSettingType

```suggestion
param powershell72Modules powershell72ModuleType
```